integ-test/src/test/java/org/opensearch/sql/calcite/standalone/MapAppendFunctionIT.java

```diff
@@ -42,10 +42,10 @@ public void testMapAppendWithNonOverlappingKeys() throws Exception {
         resultSet -> {
           Map<String, Object> result = getResultMapField(resultSet);
           assertEquals(4, result.size());
-          assertMapListValue(result, "key1", "value1");
-          assertMapListValue(result, "key2", "value2");
-          assertMapListValue(result, "key3", "value3");
-          assertMapListValue(result, "key4", "value4");
+          assertMapValue(result, "key1", "value1");
+          assertMapValue(result, "key2", "value2");
+          assertMapValue(result, "key3", "value3");
+          assertMapValue(result, "key4", "value4");
```

Comment:
Should we have a higher-level assertion helper here so we can do this all in one assert?

if this ever fails with multiple key mismatches, might not be obvious what the problem is by just looking at the first mismatch (or if the length is wrong, we end up with "expected 4 found 6" or something)